appservice/src/deploy/localGitDeploy.ts

```diff
@@ -70,11 +70,15 @@ export async function localGitDeploy(client: SiteClient, fsPath: string, context
                 const tokenSource: vscode.CancellationTokenSource = new vscode.CancellationTokenSource();
                 const token: vscode.CancellationToken = tokenSource.token;
                 try {
-                    await new Promise((resolve: () => void, reject: (error: Error) => void): void => {
-                        // for whatever reason, is '-f' exists, true or false, it still force pushes
-                        const pushOptions: git.Options = forcePush ? { '-f': true } : {};
+                    await new Promise(async (resolve: () => void, reject: (error: Error) => void): Promise<void> => {
+
+                        if (commit) {
+                            await localGit.commit('Deploy using LocalGit');
```

Comment:
Since we're trying to hide that it's local git, I'd just say something like 'Deployed via Azure App Service Extension' or something like that.